ras_party/controllers/respondent_controller.py

```diff
@@ -123,6 +125,31 @@ def delete_respondents_marked_for_deletion(session):
         session.query(BusinessRespondent).filter(BusinessRespondent.respondent_id == respondent.id).delete()
         session.query(PendingEnrolment).filter(PendingEnrolment.respondent_id == respondent.id).delete()
         session.query(Respondent).filter(Respondent.id == respondent.id).delete()
+        send_account_deletion_confirmation_email(respondent.email_address, respondent.first_name)
+
+
+def send_account_deletion_confirmation_email(email_address, name):
+    """
+    Sends email notification for account deletion confirmation.
+    :param email_address:
+    :type email_address:
```

Comment:
Can you tidy this bit up?  The 2 types aren't needed (as they're typehints in the function) and rtype can go too as this function doesn't return anything

Reply:
done